### PR TITLE
Improve except

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

File: sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def native_col_type(col_type, value):
     if col_type == "datetime":
         try:
             value = Deserializer.deserialize_iso(value)
-        except: # pylint: disable=bare-except
+        except Exception: # pylint: disable=bare-except
```
Comment (rakshith91) [inline sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py:121]:
```suggestion
        except Exception:
```

File: sdk/monitor/azure-monitor-query/CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## 1.0.2 (Unreleased)
 
-- This version and all future versions will require Python 3.7+. Python 2.7 and 3.6 are no longer supported.
+- This version and all future versions will require Python 3.6+. Python 2.7 and 3.6 are no longer supported.
```
Comment (lmazuel) [inline sdk/monitor/azure-monitor-query/CHANGELOG.md:5]:
```suggestion
- This version and all future versions will require Python 3.6+. Python 2.7 is no longer supported.
```

File: sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py [hunk 3]
```diff
@@ -118,7 +118,7 @@ def native_col_type(col_type, value):
     if col_type == "datetime":
         try:
             value = Deserializer.deserialize_iso(value)
-        except: # pylint: disable=bare-except
+        except Exception:
```
Comment (lmazuel) [inline sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py:121]: You may get a "pylint exception too broad", but this one is fine to disable in this context

Comment (rakshith91) [inline sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py:121]:
```suggestion
        except Exception: # pylint: disable=broad-except
```

Comment (rakshith91) [reply sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py]: fixed

**Review by swathipil — APPROVED** [review]

**Pull request merged**